### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by jburel**
Update readme
see https://trello.com/c/SlXTxbzz/715-secvuln-devspace

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,6 +51,9 @@ Clone infrastructure repository:
 
 NOTE: VM will boot from volume, you no longer have to attach additional volumes. Size of the volume can be set by `-e vm_size=100`
 
+    Install the various ansibles roles
```
Comment (mtbc) [inline README.md]: "ansible" maybe, but :+1: for explanations

**Comment by aleksandra-tarkowska** [comment]
There is a pr cleaning readme up in https://github.com/openmicroscopy/devspace/pull/65. If you wish any more changes please open a pr against 65.

**Comment by aleksandra-tarkowska** [comment]
any updates to doc needs to appear in readme https://github.com/openmicroscopy/ansible-role-devspace

**Comment by jburel** [comment]
Closing this PR I will open a new one against #65

**Pull request closed without merging**